console/src/routes.js

```diff
@@ -48,7 +73,11 @@ const routes = store => {
       <Route
         path=""
         component={Main}
-        onEnter={composeOnEnterHooks([requireSchema, requireMigrationStatus])}
+        onEnter={composeOnEnterHooks([
+          requireSchema,
+          requireMigrationStatus,
+          requireServerVersion,
```

Comment:
We need to move it inside the layout instead of onEnter because if if that hook doesn't succeed, then console will be down. We can leave it if this is the intended behaviour.